viewer_params_nodes:
  cam_in_body: [  1., 0. , 0., 0.,
                  0.,  1., 0., 0.,
                  0.,  0., 1., 0.,
                  0.        ,  0.        ,  0.        ,  1.        ]
  world_frame_transform: [0, 0, 1, 0,
                         -1, 0, 0, 0,
                          0, -1, 0, 1.5,
                           0, 0, 0, 1]
  camera_topic: '/camera/depth/image_rect'
  odom_topic: '/camera/pose'
  is_legacy_odom_msg: False
  is_icl_bag: True
  img_scale_factor: 0.001
  img_lag: 0.0 # Lag of the image stream vis a vis vicon stream
  view_octomap: True
  render_likelihoods: False
  rotation_thresh: 1.0
  translation_thresh: 1.0

gvdb_params:
  K: [525, 0., 319.5, 0., 525, 239.5, 0., 0., 1.]
  res: 0.05
  prior: 0.1
  dims: [30.0, 20.0, 10.0]
  occ_thresh: 0.5
  max_iters: 3
  gvdb_map_config: [3, 3, 3, 3, 3]
  rows: 480
  cols: 640
  use_polys: False
  poly_degree: 3
  depth_sigma_lookup: ""
  depth_bias_lookup: ""
  const_sigmasq_poly: [0.05, 0, 0]
  # const_sigmasq_poly: [ -0.00029, 0.00037, 0.001365]
  # const_sigmasq_poly: [0.00313, 0.00116, 0.0]
  # For kinect 1, -0.00029 + 0.00037 · z + 0.001365 . z^2
  # For kinect 2, = 0.00313 + 0.00116 · z + 0.00052 · z^2
  # Previous default: const_sigmasq_poly: [0.075, 0, 0]